SConstruct

```diff
@@ -105,7 +105,7 @@ opts.AddVariables(
     BoolVariable('ccache', "Use ccache", False),
     ('ctool', 'Set c compiler command if not using standard compiler.'),
     ('cxxtool', 'Set c++ compiler command if not using standard compiler.'),
-    EnumVariable('cxx_std', 'Target c++ std version', '11', ['11', '14', '1y']),
+    EnumVariable('cxx_std', 'Target c++ std version', '11', ['11', '14', '17']),
```

Comment:
GCC doesn't support `-std=c++17` yet, see https://gcc.gnu.org/onlinedocs/gcc-7.2.0/gcc/C-Dialect-Options.html#C-Dialect-Options

Please make the last option `1z` instead.